server/core/src/https/views/admin/accounts.rs

```diff
@@ -0,0 +1,81 @@
+use crate::https::extractors::{AccessInfo, VerifiedClientInformation};
+use crate::https::middleware::KOpId;
+use crate::https::views::errors::HtmxError;
+use crate::https::views::HtmlTemplate;
+use crate::https::ServerState;
+use askama::Template;
+use axum::extract::State;
+use axum::http::Uri;
+use axum::response::{IntoResponse, Response};
+use axum::Extension;
+use axum_htmx::{HxPushUrl, HxRequest};
+use futures_util::TryFutureExt;
+use kanidm_proto::attribute::Attribute;
+use kanidm_proto::constants::{ATTR_DISPLAYNAME, ATTR_NAME, ATTR_UUID};
+use kanidm_proto::v1::Entry;
+use kanidmd_lib::constants::EntryClass;
+use kanidmd_lib::filter::{f_eq, f_and, FC, Filter};
+use serde::{Deserialize, Serialize};
+
+#[derive(Template)]
+#[template(path = "admin/admin_overview.html")]
+struct AccountsView {
+    access_info: AccessInfo,
+    partial: AccountsPartialView,
+}
+
+#[derive(Template)]
+#[template(path = "admin/admin_accounts_partial.html")]
+struct AccountsPartialView {
+    accounts: Vec<AccountInfo>
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct AccountInfo {
+    uuid: String,
+    name: String,
+    displayname: String,
+}
+
+pub(crate) async fn view_accounts_get(
+    State(state): State<ServerState>,
+    HxRequest(is_htmx): HxRequest,
+    Extension(kopid): Extension<KOpId>,
+    VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
+) -> axum::response::Result<Response> {
+    let filter = filter_all!(f_and!([f_eq(
+        Attribute::Class,
+        EntryClass::Account.into()
+    )]));
+    let base: Vec<Entry> = state
+        .qe_r_ref
+        .handle_internalsearch(client_auth_info.clone(), filter, None, kopid.eventid)
+        .map_err(|op_err| HtmxError::new(&kopid, op_err))
+        .await?;
+
+    let accounts = base.into_iter().map(|entry: Entry| {
+        let uuid = entry.attrs.get(ATTR_UUID).unwrap_or(&vec![]).first().unwrap_or(&"".to_string()).clone();
```

Comment:
This feels really messy/hacky. We should get the entry either with a search method for scim (similar to https://github.com/kanidm/kanidm/blob/master/server/core/src/actors/v1_scim.rs#L206 ) or making a new version of https://github.com/kanidm/kanidm/blob/master/server/core/src/actors/v1_read.rs#L422 which DOES NOT call into_proto_array() so that we have the strongly typed EntrySealed version instead.

Reply:
Most likely we want the scim version because that will have pagination.